extensions/wikia/AdEngine/js/context/adContext.js

```diff
@@ -206,7 +206,8 @@ define('ext.wikia.adEngine.adContext', [
 
 		if (context.providers.rubiconFastlane) {
 			context.providers.rubiconFastlane = geo.isProperGeo(instantGlobals.wgAdDriverRubiconFastlaneCountries) &&
-				geo.isProperGeo(instantGlobals.wgAdDriverRubiconFastlaneProviderCountries);
+				geo.isProperGeo(instantGlobals.wgAdDriverRubiconFastlaneProviderCountries) &&
+				!geo.isProperGeo(instantGlobals.wgAdDriverRubiconFastlanePrebidCountries); // disable non-prebid implementation when Fastlane adapter is active
```

Comment:
Is it possible to use InstantGlobal only once? Eg. use `!context.bidders.fastlane`